lib/Images.php

```diff
@@ -7,6 +7,18 @@
 
 class Images
 {
+    private static function _getImgixConfig() {
+        $imgixDomain = getenv('CUSTOM_IMGIX_DOMAIN');
+        $imgixSignKey = getenv('CUSTOM_IMGIX_SIGN_KEY');
+
+        if ($imgixDomain && $imgixSignKey) {
+            return [
+                'domain' => $imgixDomain,
+                'signKey' => $imgixSignKey
+            ];
+        }
```

Comment:
Should we throw an error if this isn't set?

Reply:
Probably not, if it's not set then we don't use Imgix and return the original url.

Reply:
Ah yes. Ignore me.